test/handlers.test.js

```diff
@@ -128,7 +214,26 @@ describe(__filename, function () {
         expect(stubs['./http'].getCall(0).args[0]).to.deep.equal({
           guid: guid,
           method: 'DELETE',
-          path: '/' + dataset + '/' + id,
+          path: urlJoin(pathToPrepend, dataset, id),
+          timeout: 25000
+        });
+
+        done();
+      });
+    });
+  });
+
+  describe('#handleDelete', function () {
```

Comment:
That example for opts.pathToPrepend is invalid, it needs to do the urlJoin stuff, but hope it makes sense?